### PR TITLE
feat:add custom config for social widgets

### PR DESCRIPTION
**Pull request by yuqingc**
Hi,
I have implemented config for custom sicial widgets. (https://github.com/Vimux/Mainroad/issues/151#issuecomment-560962080)

Add the following to `config.toml`

```toml
[[Params.widgets.social.custom]]
  title = "Youtube"
  url = "https://youtube.com"
  icon = "youtube.svg" # Optional. Path relative to "layouts/partials"

[[Params.widgets.social.custom]]
  title = "Steam"
  url = "https://store.steampowered.com/"
  icon = "steam.svg"

[[Params.widgets.social.custom]]
  title = "My Home Page"
  url = "http://example.com"
```

and it will render this:
image

@Vimux

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,21 @@ googleAnalytics = "" # Enable Google Analytics by entering your tracking id
   gitlab = "username"
   bitbucket = "username"
   email = "example@example.com"
+
+# Your custom social links
+[[Params.widgets.social.custom]]
+  title = "Youtube"
+  url = "https://youtube.com"
+  icon = "youtube.svg" # Optional. Path relative to "layouts/partials"
```
[the remark this answers is not in the document]
Comment (yuqingc) [reply README.md]: Good idea!

**Comment by yuqingc** [comment]
@Vimux PR is updated :)

**Comment by yuqingc** [comment]
@Vimux I updated README. By the way, I have checkout the current svg file size, which turns out to be 24x24. I fixed that.

**Comment by Vimux** [comment]
LGTM, thanks.

**Comment by contributorpw** [comment]
Many thanks!

**Pull request merged**